draft-ietf-add-svcb-dns.md

```diff
@@ -76,6 +79,12 @@ This key is used to indicate the target port for connection ({{Section 6.2 of SV
 
 This key is automatically mandatory if present.  (See {{Section 7 of SVCB}} for the definition of "automatically mandatory".)
 
+Support for the `port` key can be unsafe if the client has implicit elevated access to some network service (e.g. a local service that is inaccessible to remote parties) and that service uses a TCP-based protocol other than TLS.  A hostile DNS server might be able to manipulate this service by causing the client to send a specially crafted TLS SNI or session ticket that can be misparsed as a command or exploit.  To avoid such attacks, clients SHOULD NOT support the `port` key unless one of the following conditions applies:
+
+* The client is being used with a DNS server that is highly trusted.
```

Comment:
```suggestion
* The client is being used with a trusted DNS server.
```

Trusted seems binary in this context, so "highly" doesn't seem to add anything.

Reply:
I don't think it's binary.  DNS servers have access to sensitive user data by default, among other things, so the "level of trust" is complex.

I've changed this to

> The client is being used with a DNS server that it trusts not attempt this attack.